helpers/currency-format.js

```diff
@@ -22,10 +23,24 @@ const formatCurrency = (amount, options) => {
   });
 
   if (options.isMoney) {
-    return new Intl.NumberFormat(options.locale, {
+    // Set the amount of decimals we want to show, it's 2 or 0
+    const hasTrailingZeroZero = amount === 0 || amount % 1 === 0;
+
+    if (hasTrailingZeroZero && !options.decimalTerminator) {
+      currencyStyle.maximumFractionDigits = 0;
+      currencyStyle.minimumFractionDigits = 0;
+    }
+
+    let localisedValue = new Intl.NumberFormat(options.locale, {
       style: 'currency',
       currency: options.abbreviation,
     }).format(amount);
+
+    if (hasTrailingZeroZero && options.decimalTerminator) {
+      localisedValue = `${localisedValue.slice(0, -2)}${options.decimalTerminator}`;
```

Comment:
Nice solution

Reply:
Only thing is that this would not work with money sign behind the comma/dot. But that will probably never be the case because it only for the netherlands.